README.md

```diff
@@ -30,6 +30,11 @@ New users, you don't have to worry about this - just follow the instructions in
 ### Prerequisites
 
 #### First, clone the repo to your computer, then navigate to the starfish folder and run `npm install`.
+- This project runs on **node 10.13.0 or higher**, node 12 is preferred.
+- `node -v `, will show you which version of node you're using in your machine.
```

Comment:
Per the discussion in slack, it sounds like we want to streamline this. I think the following/last 3 bullets can be replaced with something like: 
- Either update to a compatible version of node, or install nvm so that starfish can use a compatible node version without affecting other applications on your system.

Reply:
added this bullet point and a link to nvm docs